src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,9 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        return UserService.findByEmail(email) != null
```

Comment:
Check previous comment
You should create object UserService first, then 
`User user = userService.findByEmail(email);`
`return user != null && password.equals(user.getPassword());`

Reply:
How can I make the object private? 
(https://github.com/user-attachments/assets/739e040b-a0f4-4188-b155-e38282eb64b5)